designer/server/src/lib/forms.js

```diff
@@ -106,6 +115,13 @@ export function getAuthor(credentials) {
   return { id, displayName }
 }
 
+/**
+ * @param {string} token
+ */
+function getAuthOptions(token) {
```

Comment:
Good thinking this with!

Think it's worth passing the token right through to our fetch library?

Otherwise let's make sure the types don't drift

```suggestion
/**
 * @param {string} token
 * @returns {Pick<import('~/src/lib/fetch.js').RequestOptions, 'headers'>}
 */
function getAuthOptions(token) {
```

<br>

Can import the types from `~/src/lib/fetch.js`

```js
/**
 * @typedef {import('~/src/lib/fetch.js').RequestOptions} RequestOptions
 */
```

Reply:
Resolve with c6117ec826f3bed3f097779d058a1a6e99fdfcb9

> Think it's worth passing the token right through to our fetch library?

Yes - this was just to get us going. I'm assuming it'll get refactored when we do the `FM` client or `refreshToken` work.